action.yml

```
@@ -9,6 +9,8 @@ inputs:
     description: 'Run the action in dry mode (i.e. do not write optimized SVGs)'
   ignore:
     description: 'A glob of SVGs that should not be optimized'
+  strict:
+    description: 'Fail the Action run in case of a recoverable error'
```

This description could be improved, e.g.:

```suggestion
    description: 'Fail the Action run even in the event of a recoverable error'
```